README.md

```diff
@@ -1,23 +1,26 @@
 # Docker Mailserver
 
-[![ci::status]][ci::github] [![docker::pulls]][docker::hub]
+[![ci::status]][ci::github] [![docker::pulls]][docker::hub] [![documentation::badge]][documentation::web]
 
 [ci::status]: https://img.shields.io/github/workflow/status/docker-mailserver/docker-mailserver/Build%2C%20Test%20%26%20Deploy?color=blue&label=CI&logo=github&logoColor=white&style=for-the-badge
 [ci::github]: https://github.com/docker-mailserver/docker-mailserver/actions
 [docker::pulls]: https://img.shields.io/docker/pulls/mailserver/docker-mailserver.svg?style=for-the-badge&logo=docker&logoColor=white
 [docker::hub]: https://hub.docker.com/r/mailserver/docker-mailserver/
+[documentation::badge]: https://img.shields.io/badge/DOCUMENTATION-GH%20PAGES-0078D4?style=for-the-badge&logo=git&logoColor=white
+[documentation::web]: https://docker-mailserver.github.io/docker-mailserver/edge/
 
-A fullstack but simple mail server (SMTP, IMAP, LDAP, Antispam, Antivirus, etc.). Only configuration files, no SQL database. Keep it simple and versioned. Easy to deploy and upgrade.
+A fullstack but simple mail server (SMTP, IMAP, LDAP, Antispam, Antivirus, etc.). Only configuration files, no SQL database. Keep it simple and versioned. Easy to deploy and upgrade. [Documentation][documentation::web] via MkDocs. [Why this image was created.](http://tvi.al/simple-mail-server-with-docker/).
 
-[Why this image was created.](http://tvi.al/simple-mail-server-with-docker/)
+If you have issues, read the full `README` **and** the [documentation][documentation::web] **for your version** first **before opening an issue**. The issue tracker is for issues, not for personal support.
```

Comment:
I have included the

> `(default is edge)`

---

As for GH Discussions: If you ask me, **no way** :D This will lead to a situation where everyone will come and ask because something in his/her setup doesn't work, and they do not know how to use a search engine:)